src/Database/Expression/CaseExpression.php

```diff
@@ -130,6 +130,10 @@ protected function _addExpressions($conditions, $values, $types)
                 $value = $keyValues[$k];
                 array_push($this->_values, $value);
                 continue;
+            } elseif ($value === 'identifier') {
```

Comment:
use of elseif is more compact. other reason i sued it there already in next lines
`
                continue;
            } elseif ($value instanceof ExpressionInterface) {
`